DGL_testing/gpu_test.py

```diff
@@ -0,0 +1,37 @@
+import dgl_ptm
```

Comment:
can we have this test file in the `tests` folder instead?

Reply:
I renamed it as a sample run as it is not exactly a test or formatted as such. Now it is called 'gpu_sample_run.py'